src/mscorlib/shared/System/Buffers/TlsOverPerCoreLockedStacksArrayPool.cs

```diff
@@ -254,13 +381,24 @@ public T[] TryPop()
                 }
                 return null;
             }
+
+            public bool Trim(uint tickCount, int id, MemoryPressure pressure)
+            {
+                LockedStack[] stacks = _perCoreStacks;
+                for (int i = 0; i < stacks.Length; i++)
+                {
+                    stacks[i].Trim(tickCount, id, pressure);
+                }
+                return true;
+            }
         }
 
         /// <summary>Provides a simple stack of arrays, protected by a lock.</summary>
         private sealed class LockedStack
         {
             private readonly T[][] _arrays = new T[MaxBuffersPerArraySizePerCore][];
             private int _count;
+            private uint _stockTicks;
```

Comment:
CAn you think of a better name. This makes me think of stock tickers. `nextActionTicks` ?

Reply:
`_firstStackItemMS`